docs/sphinx/source/fsd/FSD-Appendices.md

```diff
@@ -0,0 +1,30 @@
+# Appendix A
+
+See spreadsheet named [Appendix A.xls](../Appendix%20A.xls).
+
+# Appendix B
+
+## Metadata Profile Definitions
+
+### NOAA Office of Coast Survey Metadata profile, October 2022 (OCS-2022-10)
+
+| Column Name | Column Type | Note |
+| --- | --- | --- |
+| significant_features | Boolean | See S-101 significant features detected. |
+| feature\_least\_depth | Boolean | See S-101 least depth of detected feature measured. |
+| feature_size | Float | See S-101 feature size. |
+| feature\_size\_var | Float | See further discussion (1) |
+| coverage | Boolean | See S-101 full seafloor coverage achieved |
+| bathy_coverage | Boolean | See further discussion (2) |
+| horizontal\_uncert\_fixed | Float | See S-101 horizontal position uncertainty fixed |
+| horizontal\_uncert\_var | Float | See S-101 horizontal position uncertainty variable factor |
+| survey\_date\_start | String | See S-101 Survey date start |
+| survey\_date\_end | String | See S-101 Survey date end |
+| Source Institution | String | e.g. "NOAA Office of Coast Survey" |
```

Comment:
@GlenRice-NOAA Is it okay if I modify the all column names to be in snake case, without spaces?